app/src/lib/date-utils.ts

```diff
@@ -36,3 +36,20 @@ export function getTodayUtcDate(): string {
 
   return todayBucket
 }
+
+export function getHourFromUtcDate(date: string): string {
+  dayjs.extend(dayJsutcPlugin)
+
+  const time = dayjs(date)
+
+  return time.format('HH:mm')
+}
+
+// Returns today's date in the format of `DD/MM/YYYY`
+export function getTodayStringTime(): string {
+  dayjs.extend(dayJsutcPlugin)
+
+  const time = dayjs()
+
+  return time.format('DD/MM/YYYY')
```

Comment:
I'd extract these formats into constants, e.g for this one: `const ISO_DATE_FORMAT = 'DD/MM/YYYY'`